include/triggeralgs/ChannelAdjacency/TriggerActivityMakerChannelAdjacency.hpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file TriggerActivityMakerChannelAdjacency.hpp
+ *
+ * This is part of the DUNE DAQ Application Framework, copyright 2021.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#ifndef TRIGGERALGS_CHANNELADJACENCY_TRIGGERACTIVITYMAKERCHANNELADJACENCY_HPP_
+#define TRIGGERALGS_CHANNELADJACENCY_TRIGGERACTIVITYMAKERCHANNELADJACENCY_HPP_
+
+#include "triggeralgs/TPWindow.hpp"
+#include "triggeralgs/TriggerActivityFactory.hpp"
+#include <fstream>
+#include <vector>
+
+namespace triggeralgs {
+class TriggerActivityMakerChannelAdjacency : public TriggerActivityMaker
+{
+public:
+  void operator()(const TriggerPrimitive& input_tp, std::vector<TriggerActivity>& output_ta);
+  void configure(const nlohmann::json& config);
+
+private:
+  TriggerActivity construct_ta(TPWindow) const;
+
+  TPWindow check_adjacency();
+
+  TPWindow m_current_window;              // Holds collection hits only
```

Comment:
It does not look like the collection hits claim is true.

Reply:
removed now